.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+# Notes:
+#  - All software installed below is pinned so they don't get upgraded and break w/o us knowing why.
+#  - If you bump one version up to latest, please bump them all so we stay current!
+
+name: Build medic-couch2pg and run unit test cases
+
+on: [push, pull_request]
+
+jobs:
+  build-and-test:
+    name: Build medic-couch2pg
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+
+    - name: Use nodeJS 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
```

Comment:
The comment at the top notes the "pinned" management of the dependencies. The `.x` here doesn't seem to match that comment. Is the desire for this to dynamically pick up the latest from version 10? If so, that seems fine, but the comment at the top would need to match. If we do want a specific version though, let's just specify that here.... or does this step input not allow for that?

Reply:
I've fixed the build in node versions in this [PR](https://github.com/medic/medic-couch2pg/pull/73), code adapted to test in version 8, 10 and 12.